Source/bmalloc/libpas/libpas.xcodeproj/project.pbxproj

```diff
@@ -2952,7 +2952,7 @@
 				MACOSX_DEPLOYMENT_TARGET = 11.0;
 				MTL_FAST_MATH = YES;
 				PRODUCT_NAME = "$(TARGET_NAME)";
-				SDKROOT = macosx;
+				SDKROOT = macosx.internal;
```

Comment:
Should we use some mechanism to switch them instead of always using `internal`? (Like, what we are doing for the rest of the project).

Reply:
Probably would be worth doing in the future, but you can build from the command line using Xcode build, and we have a cmake build which works on macOS.

Currently, this isn't going to run properly on macOS for either SDK.